client/src/hooks/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./useHash";
```

Comment:
This is probably the beginning of a barrel file: https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/ Let's try to avoid them.